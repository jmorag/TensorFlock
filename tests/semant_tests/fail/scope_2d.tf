main = f 42;

f : Nat -> Nat;
f n = n + b + g; {
    b : Nat;
    b = g; {
        g : Nat;
        g = 24;
   }
}
