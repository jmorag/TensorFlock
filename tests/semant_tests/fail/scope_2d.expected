Fatal error: exception Failure("Encountered undefined symbol: g")
